### PR TITLE
Snap 2986 : Refactor scripts for TPC-H smart connector and TPC-DS

### PR DESCRIPTION
**Pull request by cruizen**
## Changes proposed in this pull request

Refactor scripts for TPC-H smart connector and TPC-DS 

## Patch testing

Manual and TPCHDUnit

## ReleaseNotes.txt changes

(Does this change require an entry in ReleaseNotes.txt? If yes, has it been added to it?)
No 

## Other PRs 

(Does this change require changes in other projects- store, spark, spark-jobserver, aqp? Add the links of PR of the other subprojects that are related to this change)

### CODE REVIEW

File: tests/benchmark/snappy/dynamic/smartConnector/PerfRun.conf.template [hunk 1]
```diff
@@ -18,44 +24,38 @@ locatorProperties="-conserve-sockets=false"
 serverProperties="-heap-size=3g -memory-size=3g -conserve-sockets=false"
 
 #Machine Setup
-master=localhost
+master=tkhairnar-dell
```
Comment (swatisawant) [inline tests/benchmark/snappy/dynamic/smartConnector/PerfRun.conf.template]: Just add a description in the comment ahead of hostname explaining the issues with specifying the master hostname as localhost.

Comment (cruizen) [reply tests/benchmark/snappy/dynamic/smartConnector/PerfRun.conf.template]: Okay.

File: tests/benchmark/snappy/dynamic/smartConnector/5_createAndLoadTable.sh [hunk 2]
```diff
@@ -2,5 +2,17 @@
 
 source PerfRun.conf
 
+echo "bash $SPARK_HOME/bin/spark-submit --master spark://$master:7077 \
+--conf spark.snappydata.connection=$locator:1527 \
+$sparkProperties --class io.snappydata.benchmark.snappy.tpch.TableCreationSmartConnector \
+$TPCHJar $dataDir $NumberOfLoadStages $Parquet \
+$buckets_Order_Lineitem $buckets_Cust_Part_PartSupp $IsSupplierColumnTable \
+$buckets_Supplier $Redundancy $Persistence $Persistence_Type 1 "
```
[the remark this answers is not in the document]
Comment (cruizen) [reply tests/benchmark/snappy/dynamic/smartConnector/5_createAndLoadTable.sh]: Yes, added a conf variable for better readability.

File: tests/benchmark/snappy/dynamic/smartConnector/6_executeQuery.sh [hunk 3]
```diff
@@ -3,5 +3,9 @@
 source PerfRun.conf
 
 #Execute Spark App
-sh $SPARK_HOME/bin/spark-submit --master spark://$master:7077 --conf spark.snappydata.connection=$locator:1527 $sparkProperties --class io.snappydata.benchmark.snappy.tpch.QueryExecutionSmartConnector $TPCHJar $queries $sparkSqlProperties $IsDynamic $ResultCollection $WarmupRuns $AverageRuns 1
+bash $SPARK_HOME/bin/spark-submit \
+--master spark://$master:7077 \
+--conf spark.snappydata.connection=$locator:1527 \
+$sparkProperties --class io.snappydata.benchmark.snappy.tpch.QueryExecutionSmartConnector \
+$TPCHJar $queries $sparkSqlProperties $IsDynamic $ResultCollection $WarmupRuns $AverageRuns 1
```
[the remark this answers is not in the document]
Comment (cruizen) [reply tests/benchmark/snappy/dynamic/smartConnector/6_executeQuery.sh]: Added a threadNumber conf variable.

**Pull request merged**